src/Umbraco.Infrastructure/Migrations/Upgrade/V_9_0_0/ExternalLoginTableIndexes.cs

```diff
@@ -45,8 +45,8 @@ protected override void Migrate()
                 Create
                      .Index(indexName1)
                      .OnTable(ExternalLoginDto.TableName)
-                     .OnColumn("loginProvider")
-                     .Ascending()
+                     .OnColumn("loginProvider").Ascending()
```

Comment:
we'll actually need to make a new migration for this so that current sites using the rc get updated, but i'll fix that up.